climada/hazard/storm_europe.py

```diff
@@ -830,3 +831,106 @@ def _hist2prob(self, intensity1d, sel_cen, spatial_shift, ssi_args=None,
+                             haz_model = 'icon-eu-eps',
+                             haz_raw_storage = None):
+    """ use parameters to generate needed forecast hazard either by download
+    or through file.
+    Parameters:
+    run_datetime (datetime, optional): The starting timepoint of the forecast run
+        of the icon model, defaults to today 00:00 hours
+    event_date (datetime, optional): one day within the forecast
+        period, only this day (00H-24H) will be included in the hazard,
+        defaults to the day after tomorrow
+    haz_model (str,optional): select the name of the model to
+        be used: one of ['icon-eu-eps', 'cosmo1e_file', 'cosmo2e_file']
+    haz_raw_storage (str, optional): path to folder, where netcdf files
+        are stored, mendatory when haz_model is 'cosmo1e_file' or
+        'cosmo2e_file'. the string must contain one set of curly brakets,
+        used by the function str.format to include
+        run_datetime.strftime('%y%m%d%H').
+    Returns:
+        hazard
+        haz_model
+        run_datetime
+        event_date
+    """
+    if (haz_model == 'cosmo1e_file'
+        or
+        haz_model == 'cosmo2e_file'):
+        if haz_model == 'cosmo1e_file':
+            haz_model='C1E'
+            full_model_name_temp = 'COSMO-1E'
+        if haz_model == 'cosmo2e_file':
+            haz_model='C2E'
+            full_model_name_temp = 'COSMO-2E'
+        # haz_file_name = Path(haz_dir) / (haz_summary_str +
+        #                                       '.hdf5')   
+        # if haz_file_name.exists():
+        #     LOGGER.info('Loading hazard from ' + 
+        #                 str(haz_file_name) + 
+        #                 '.')
+        #     hazard = StormEurope()
+        #     hazard.read_hdf5(haz_file_name)
+        # else:
+        LOGGER.info('Generating ' + 
+                    haz_model + 
+                    ' hazard.')  
+        if not haz_raw_storage:
+            haz_raw_storage = (
+                "D:\\Documents_DATA\\Cosmo\\Wind\\" +
+                "cosmoe_forecast_{}_vmax.nc"
+                )
+        fp_file = Path(
+            haz_raw_storage.format(
+                run_datetime.strftime('%y%m%d%H')
+                )
+            )
+        hazard = StormEurope()
+        hazard.read_cosmoe_file(
+            fp_file, 
+            event_date=event_date,
+            run_datetime=run_datetime,
+            model_name=full_model_name_temp
+            )
+    elif haz_model == 'icon-eu-eps':
+        haz_model='IEE'
+        # haz_file_name = Path(haz_dir) / (haz_summary_str +
+        #                                       '.hdf5')
+        # if haz_file_name.exists():
+        #     LOGGER.info('Loading hazard from ' + 
+        #                 str(haz_file_name) + 
+        #                 '.')
+        #     hazard = StormEurope()
+        #     hazard.read_hdf5(haz_file_name)
+        # else:
```

Comment:
?

Reply:
uncommented again after adding the folder to CONFIG